app/controllers/avo/base_controller.rb

```diff
@@ -63,8 +63,9 @@ def index
 
       # Create resources for each record
       # Duplicate the @resource before hydration to avoid @resource keeping last record.
+      @resource.hydrate(params: params)
       @resources = @records.map do |record|
-        @resource.dup.hydrate(record: record, params: params)
+        @resource.dup.hydrate(record: record)
```

Comment:
Unnecessary to assign params to each duplicate, we can assign once to the original one.